src/libs/actions/Report.js

```diff
@@ -1269,6 +1269,29 @@ function saveReportActionDraft(reportID, reportActionID, draftMessage) {
     Onyx.set(`${ONYXKEYS.COLLECTION.REPORT_ACTIONS_DRAFTS}${reportID}_${reportActionID}`, draftMessage);
 }
 
+/**
+ * Syncs up a chat report and an IOU report in Onyx after an IOU transaction has been made
+ * and they've both been updated in the back-end.
+ *
+ * @param {Object} chatReportStuff
+ * @param {Object} iouReportStuff
+ */
+function syncChatAndIOUReports(chatReportStuff, iouReportStuff) {
```

Comment:
What happens if one of the reports is empty, due to a backend issue? Could we check for null reports and return early as a basic protection?

Reply:
I worry that in this case we'll be putting bad data into Onyx.